Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -248,20 +261,22 @@ public void centerConeStack(RobotCameraPipeline pipeline) {
         double xOffset, width;
 
         do {
-            xOffset = pipeline.xPosition - Constants.CAMERA_CENTER_X;
+            xOffset = 400-pipeline.xPosition;
```

Comment:
Where is this 400 coming from? This should also be a variable with a useful name

Reply:
Resolved, added back the old constant